neuraxle/base.py

```diff
@@ -1346,10 +1346,9 @@ def fit_transform(self, data_inputs, expected_outputs=None):
         * :class:`StepClonerForEachDataInput`
     """
 
-    # TODO: remove equal None, and fix random search at the same time ?
     def __init__(
             self,
-            wrapped: BaseStep = None
+            wrapped: BaseStep
```

Comment:
at least make it optionally equal to None to reserve the right to call just `.set_step` later?